src/recommendpolicy/recommendPolicy.go

```diff
@@ -135,6 +142,16 @@ func RecommendPolicyMain() {
 		log.Error().Msg(err.Error())
 		return
 	}
+	replicaSets, err := client.AppsV1().ReplicaSets("").List(context.Background(), metav1.ListOptions{})
```

Comment:
@Prateeknandle When we get all the replicasets there is a possibility that the replicaset was created by a deployment. If that is the case then we will receive duplicate policies for the same. 
For eg., If we deploy wordpress-mysql application, we can see that it'll create replicasets from the deployment. 
```
k -n wordpress-mysql get deploy,rs                                                                                  (get_hardening_pol↓1|✚1)
NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/mysql       0/1     1            0           6d3h
deployment.apps/wordpress   1/1     1            1           6d3h

NAME                                   DESIRED   CURRENT   READY   AGE
replicaset.apps/mysql-76ddc6ddc4       1         1         0       6d3h
replicaset.apps/wordpress-787f45786f   1         1         1       6d3h
```
Naturally we wouldn't want policies to be generated for both (ie, duplicate) Can we handle this scenario as well?

Reply:
Handled in new commit